src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/MetaType.php

```diff
@@ -112,13 +120,28 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                                 'Admin.Notifications.Error'
                             ),
                         ]),
+                        new Length([
+                            'max' => SeoSettings::MAX_TITLE_LENGTH,
+                            'maxMessage' => $this->trans(
+                                'This field cannot be longer than %limit% characters',
+                                [
+                                    '%limit%' => SeoSettings::MAX_TITLE_LENGTH,
+                                ],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
                     ],
-                    'required' => false,
```

Comment:
Do not remove this one.

Reply:
Moved to https://github.com/PrestaShop/PrestaShop/pull/14469/files#diff-46dd18c389c6386f6ad082fce76cfb79R108